app/src/main/java/org/simple/clinic/widgets/qrcodescanner/QrCodeScannerView.kt

```diff
@@ -59,6 +61,9 @@ constructor(
   @Inject
   lateinit var bitmapUtils: BitmapUtils
 
+  @Inject
+  lateinit var googleApiAvailability: GoogleApiAvailability
```

Comment:
Correct me if I am wrong, but this will only tell us if Play Services is available, right? Not whether the MLKit dependencies are downloaded or not.

Reply:
Yep, we check if the ML Kit is available from the crash in the `MlKitQrCodeAnalyzer`